roles/libvirt_manager/tasks/deploy_layout.yml

```diff
@@ -226,13 +270,23 @@
         '(cifmw-)?' ~
         '([a-z0-9]+)-[0-9]+'
       }}
+    _ocp_master_matcher: >-
+      {{
+        '^ocp-(\d+)$'
+      }}
     vm_type: "{{ vm | regex_replace(_matcher, '\\2') }}"
     vm_data: >-
       {{
         _cifmw_libvirt_manager_layout.vms[vm_type]
       }}
     pub_key: "{{ pub_ssh_key.content | b64decode }}"
     priv_key: "{{ priv_ssh_key.content | b64decode }}"
+    # If $vm is an OCP master node, capture it's index
+    # and craft the _vm_img varible, examples:
+    # ocp-0 -> ocp_master-0.qcow (match and modify)
+    # ocp_worker-0 -> ocp_worker-0.qcow (pass)
+    # compute-1 -> compute-1.qcow (pass)
+    _vm_img: "{{ vm | regex_search(_ocp_master_matcher) | ternary(vm | regex_replace(_ocp_master_matcher, 'ocp_master-\\1.qcow2'), vm ~ '.qcow2') }}"
```

Comment:
Not sure this is really needed? the disk_image_name is supposed to be `ocp_master` already. So I'd expect it to be "easy" to build onto that... ? Extracting the ID is needed indeed - but for the rest... I might be easier.